examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Add a comment stating what this example actually does and some additional information on getting started. you can see what other have done.